src/scriptdev2/scripts/eastern_kingdoms/naxxramas/boss_gluth.cpp

```diff
@@ -150,7 +150,9 @@ struct boss_gluthAI : public ScriptedAI
 
                 // Devour a Zombie
                 if (pZombie->IsWithinDistInMap(m_creature, 15.0f))
+                    m_creature->SetFacingToObject(pZombie);
                     m_creature->DealDamage(pZombie, pZombie->GetHealth(), nullptr, DIRECT_DAMAGE, SPELL_SCHOOL_MASK_NORMAL, nullptr, false);
+                    m_creature->SetFacingToObject(m_creature->getVictim());
```

Comment:
Alright, just followed the notes at the top, if this has been sorting corewise I'll delete it.